src/ResourceManager/IotHub/Commands.IotHub/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.IotHub" version="1.1.2" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.IotHub" version="1.1.3" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.10" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net452" />
 </packages>
```

Comment:
Please remove the psm1 files below - they are sometimes updated in the build but should not be pushed with the PR.  Also, what is the nuget package in the LocalFeed folder? Why do you need this?

Reply:
Removed nuget package from localfeed